### ISSUE TITLE
Replacement album/artist/track artwork API

### ISSUE THREAD

**Issue by jaedb** [jaedb/Iris#403]
**Is your feature request related to a problem? Please describe.**
Artwork sourced from the LastFM API no longer works (see [blog post](https://getsatisfaction.com/lastfm/topics/api-announcement-dac8oefw5vrxq)). This means artist and album artwork is replaced by a [placeholder star image](https://lastfm-img2.akamaized.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png).

**Describe the solution you'd like**
Replace the current LastFM API with something that supports artwork requests.

**Describe alternatives you've considered**
FreeCovers
Discogs
Musicbrainz

**Other considerations**
Ideally we wouldn't be dependent on more APIs than is necessary. If we can use an API that provides artwork as well as track, artist and album info then we can remove LastFM API completely which would be preferable.

**Comment by fmarzocca** [comment]
Just a thought on Musicbrainz...

You can get the artist ID by parsing this url:
`http://musicbrainz.org/ws/2/artist/?query=artist:freddie%20mercury`

the ID field is:
`artist id="022589ac-7177-460d-a178-9976cf70e29f"`

then, calling the specific artist url:
`http://musicbrainz.org/ws/2/artist/022589ac-7177-460d-a178-9976cf70e29f?inc=url-rels`

you could find the image:
```
<target id="0457caf4-9368-4339-a588-67a8f939af83">
https://commons.wikimedia.org/wiki/File:Freddie_Mercury_performing_in_New_Haven,_CT,_November_1978.jpg
</target>
```

which brings to:
![Freddie_Mercury_performing_in_New_Haven,_CT,_November_1977](https://user-images.githubusercontent.com/8530030/57917780-b265ff80-7895-11e9-8aeb-3b76a6f28707.jpg)

**Comment by fmarzocca** [comment]
With `musicbrainzngs` library you can retrieve all kinds of music metadata from the MusicBrainz database.
[(https://python-musicbrainzngs.readthedocs.io/en/v0.6/)]

**Comment by jaedb** [comment]
Options include:
- Musicbrainz (as above)
  - Seems to have very sparse results
  - `mbid` as provided by _LastFM_ seems to be a low-quality match
- Discogs
- Muzooka

**Comment by fmarzocca** [comment]
That's fair enough! Thank you James.

**Comment by jaedb** [comment]
Released at `3.41`
